api/proto/common/v2/common_v2.proto

```diff
@@ -71,8 +71,14 @@ message PaymentHeader {
   // The account ID of the disperser client. This account ID is an eth wallet address of the user,
   // corresponding to the key used by the client to sign the BlobHeader.
   string account_id = 1;
-  // The reservation period of the dispersal request. 
+  // The reservation period of the dispersal request. The dispersal client's account will set this value to 
+  // current timestamp divided by the on-chain configured reservation period interval. The disperser server validates
+  // this value to be the period they receive this request, or from the previous period.
   uint32 reservation_period = 2;
-  // The cumulative payment of the dispersal request. This field will be parsed as a big integer.
+  // The cumulative payment of the dispersal request. This field will be parsed as a big integer, and must represent
+  // the total amount of tokens paid over all previous requests. The disperser server validates this value against the 
+  // total on-chain deposit of the user's account, and the total amount of tokens paid over all previous requests. This 
+  // is not incremental but cumulative so the requests can arrive out of order but still has clear declaration of the 
+  // payment being used for the current request.
```

Comment:
not super clear what this means. adding a simple example would help.

Reply:
added more explanations and an example

Reply:
thanks, lgtm